src/ImageSearchModal.jsx

```diff
@@ -42,7 +42,7 @@ export const ImageSearchModal = ({ downloadImage, users }) => {
     const { registries } = usePodmanInfo();
     const Dialogs = useDialogs();
     // Registries to use for searching
-    const searchRegistries = registries.search && registries.length !== 0 ? registries.search : fallbackRegistries;
+    const searchRegistries = registries?.search && registries.search.length !== 0 ? registries.search : fallbackRegistries;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2219-12ebb26b-20250806-090255-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)